src/Main.hs

```diff
@@ -112,7 +125,8 @@ calcPixelAt cam world options (Z :. x :. y) =
     let (_, ny) = size cam
         (red, green, blue) = aaSample cam world x (ny - y) options
     -- convert to an "integer" (will get rounded properly later)
-    in (65534.99*(sqrt red), 65534.99*(sqrt green), 65534.99*(sqrt blue))
+    in (65534.99*(red ** (1.0 / (gamma options))), 65534.99*(green ** (1.0/(gamma options))), 65534.99*(blue ** (1.0/(gamma options))) )
```

Comment:
I think you should be able to use the scalar-vector multiplication operator I implemented (`.*`).  Also you can use `1` instead of `1.0` -- Haskell will figure the type out.